schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/OrgMemberChecker.java

```diff
@@ -39,8 +46,8 @@ protected OrgMemberChecker initialValue() {
             return new OrgMemberChecker() {
                 @Override
                 Organization getOrganization() throws EntityNotFoundException {
-                    Activity activity = ActivityServiceFactory.getActivityService().getActivity(id);
-                    ActivityType activityType = activity.getActivityTypeRef().getModel();
+                    Activity activity = getActivity(id);
+                    ActivityType activityType = getActivityType(activity.getActivityTypeRef().getKey());
                     return getOrganization(activityType.getOrganizationRef().getKey());
```

Comment:
NullPointer ?  `if(activity==null) return null;`

Same for all getActivity, getActivityType, etc